app/assets/javascripts/Components/Result/file_viewer.jsx

```diff
@@ -86,6 +88,8 @@ export class FileViewer extends React.Component {
     };
     if (this.state.loading) {
       return I18n.t('working');
+    } else if (this.state.type === null) {
```

Comment:
You can remove this (and the strings). There's already a message that should be displayed at `submission_file_panel.jsx:218`, but it seems that it's not being displayed properly. Please fix the condition in that file to make that message display in the selector.